src/lib.rs

```diff
@@ -891,6 +894,36 @@ impl Node {
 		))
 	}
 
+	/// Returns a payment handler allowing to send and receive on-chain payments.
+	#[cfg(not(feature = "uniffi"))]
+	pub fn payjoin_payment(&self) -> Result<PayjoinPayment, Error> {
+		self.payjoin_sender.as_ref().map_or_else(
+			|| Err(Error::PayjoinSenderUnavailable),
```

Comment:
All of the user payments return a struct while this returns `Result`.. I was not sure how to avoid this here

Reply:
I think changing the `sender` property in `PayjoinPayment` to `Option<..>` would work. we then can add `receiver` also as `Option<..>` and in the `send`/`receive` functions we can check if `Option` is some otherwise return `SenderUnavailable` or `ReceiverUnavailable` accordingly